docker/package/wizard_structure.py

```diff
@@ -245,9 +245,9 @@ def url_is_reachable(url):
 
 networks = {
     "mainnet": "Main Tezos network",
-    "jakartanet": "Test network using version 013 of Tezos protocol (Jakarta2)",
-    "ghostnet": "Long running test network using version 013 of Tezos protocol (Jakarta2)",
+    "ghostnet": "Long running test network using version 014 of Tezos protocol (Kathmandu)",
     "kathmandunet": "Test network using version 014 of Tezos protocol (Kathmandu)",
+    "limanet": "Long running test network using version 015 of Tezos protocol (Lima)",
```

Comment:
I would suggest dropping the protocol numbers from here too, to avoid confusion:
```suggestion
    "ghostnet": "Long running test network, currently using the kathmandu Tezos protocol",
    "kathmandunet": "Test network using the kathmandu Tezos protocol",
    "limanet": "Test network using the lima Tezos protocol",
```